lib/beaker/host/unix/exec.rb

```diff
@@ -233,15 +233,8 @@ def ssh_permit_user_environment
   def environment_string env
     return '' if env.empty?
     env_array = self.environment_variable_string_pair_array( env )
-
     environment_string = env_array.join(' ')
-    command = 'env'
-    punctuation = ''
-    if self[:platform] =~ /cisco-5/
-      command = 'export'
-      punctuation = ';'
-    end
-    "#{command} #{environment_string}#{punctuation}"
+    "env #{environment_string}"
```

Comment:
the previous code used 'export' when platform is cisco-5, presumably because 'env' cannot be used to load variables? I have no idea if that's the case but this env vs export logic is not here